html/lesson5/tutorial.md

```diff
@@ -158,8 +158,6 @@ Great! Now our images and links are on the left, and the main content on the rig
 
 [Border radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius) enables us to create rounded corners for our elements. In the past, multiple divs and the use of images was required to achieve that effect.
 
-`border-radius: arc`
```

Comment:
You are removing this example when above we explain what border radius is, can we keep it in please.